Drakhtar/State.cpp

```diff
@@ -65,11 +66,8 @@ void State::_render() const
 
 	// Render each game object
 	for (auto gameObject : gameObjects_)
-	 	gameObject->render();
-
-	exampleDialog_->render();
-
-	
+		if(gameObject->getDestroyed() == false)
```

Comment:
This should use the negation operator, also, add a space between the `if` and the `(`. Same for the next occurrence (line 92):
```suggestion
		if (!gameObject->getDestroyed())
```

Reply:
Additionally, `getDestroyed()` should be renamed to `isDestroyed()`.